### PR TITLE
Make feral humans sometimes drop their weapons on death.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Make feral humans drop their weapon on death"

#### Purpose of change

A simple flavour change: feral humans are described as wielding weapons in their IDs and in tilesets, but never drop their weapons.

#### Describe the solution

Added a chance for feral humans to drop their weapons on death. Not 100%, though.

#### Describe alternatives you've considered

Not doing so. 

#### Testing

Kill a feral zombie with these changes applied.

#### Additional context

I have NO IDEA why this wasn't implemented sooner.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
Maybe add a brick (with some chance) to the list of dropped items?

**Comment by Xenomorph-III** [comment]
Not any more! I changed the attack description to throwing rocks and added "drops_ammo" to give rocks on death.

**Comment by Maleclypse** [comment]
While you are in there do you mind removing the `NO_AMMO` flag and fixing #42169?

**Comment by Xenomorph-III** [comment]
Sure! Lemme know where the monsterguns file is... tried finding it myself, no dice.

**Comment by Xenomorph-III** [comment]
OK, @Maleclypse , the removal of "no_ammo" is done!

**Comment by Wishbringer** [comment]
Use this to lint with for better results: http://dev.narc.ro/cataclysm/format.html

File: data/json/items/gun/monster_gun.json [hunk 1]
```diff
@@ -1,110 +1,85 @@
...
+      "WATERPROOF_GUN",
+      "NO_SALVAGE",
+      "NO_UNLOAD",
+      "RELOAD_AND_SHOOT"
+    ],
+    "skill": "pistol",
+    "ammo": [ "barb" ],
+    "clip_size": 1,
+    "weight": "540 g",
+    "volume": "750ml",
```
Comment (Wishbringer) [inline data/json/items/gun/monster_gun.json]:
```suggestion
    "volume": "750 ml",
```
Applies to other "volume" instances in the file as well.

Comment (Wishbringer) [inline data/json/items/gun/monster_gun.json]:
```suggestion
    "volume": "750 ml",
```
Applies to other "volume" instances in the file as well.

Comment (Xenomorph-III) [reply data/json/items/gun/monster_gun.json]: It's done!
I'm gonna close this now.

File: data/json/monsterdrops/feral_humans.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "feral_humans_death_drops_axe",
+    "entries": [ { "item": "fire_axe", "prob": 100, "damage": [ 2, 4 ] }, { "group": "default_zombie_clothes", "prob": 100 } ]
```
Comment (AMurkin) [inline data/json/monsterdrops/feral_humans.json:6]: It looks like it should be `fire_ax`.
https://github.com/CleverRaven/Cataclysm-DDA/blob/12ea5d237de0a4ace996bc8a857eee47a0848edf/data/json/items/tool/firefighting.json#L55-L58

**Comment by jeremyshannon** [comment]
> What is this stupid shit?

I just assumed it was an inside joke of some kind. Whatever it is, there's no reason to get so riled up over a commit title.

**Pull request merged**